jest.config.js

```diff
@@ -30,11 +29,10 @@ module.exports = {
     },
   },
   transformIgnorePatterns: ['node_modules/'],
-  resolver: '<rootDir>/.jest/resolver.js',
   transform: {
-    '^.+\\.tsx?$': '<rootDir>/.jest/transform-typescript.js',
-    '^.+\\.js?$': '<rootDir>/.jest/transform-esm.js',
+    '^.+\\.(js|jsx|ts|tsx)?$': '@swc/jest',
```

Comment:
Whereas before we needed different transforms for js and ts, because they needed different babel configs, just using swc as the compiler not only means it's faster to run jest than babel, but it'll transform both source file types with no config switching. Much simpler.